app/views/markets/search_results.html.erb

```diff
@@ -0,0 +1,36 @@
+<p>
```

Comment:
This is redundant.  Change the search method in the markets_controller to render :markets and DRY.